dashboard/src/modules/components/AuthComponent/common-components.jsx

```diff
@@ -129,18 +129,16 @@ export const PasswordConstraints = (props) => {
       <h4>Passwords must contain at least:</h4>
       <div className="contraints-container">
         {passwordConstraintsText.map((constraint, index) => {
-          let variant = checkConstraints[constraint.name];
+          const variant = checkConstraints[constraint.name];
           return (
             <HelperText key={index}>
               <HelperTextItem
                 name={constraint.name}
                 variant={checkConstraints[constraint.name]}
                 icon={iconList[variant]}
               >
-                {
-                  constraint.name === "passwordLength" &&
-                  passwordLength
-                }{" "}{constraint.label}
+                {constraint.name === "passwordLength" && passwordLength}{" "}
+                {constraint.label}
               </HelperTextItem>
             </HelperText>
           );
```

Comment:
At line 137, should we be using the local variable, `variant`, that we defined at line 132 (and which we use at line 138)?

Also, what do lines 140-141 actually do?  They look to me like they produce a boolean value (possibly as a string), followed by a space character, followed by a string...what does the `<HelperTextItem>` tag do with that text?

Reply:
Lines 140-141:
This is with the UI. 

All other constraints are hard-coded except for the password length. It gets the value from redux store. It provides the `space` between `constraint.name` and `constraint.label`